receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -421,8 +523,16 @@ func TestGetRepos(t *testing.T) {
 			client := graphql.NewClient(server.URL, ghs.client)
 
 			_, count, err := ghs.getRepos(context.Background(), client, "fake query")
+			//sn, count, err := ghs.getRepos(context.Background(), client, "fake query")
 
-			assert.Equal(t, tc.expected, count)
+			//var totalVulnerabilities int
+			//for _, v := range sn {
+			//	assert.Equal(t, len(v.VulnerabilityAlerts.Nodes), v.VulnerabilityAlerts.TotalCount)
+			//	totalVulnerabilities += len(v.VulnerabilityAlerts.Nodes)
+			//}
+
+			//assert.Equal(t, tc.expectedVulnerabilities, totalVulnerabilities)
+			assert.Equal(t, tc.expectedRepos, count)
```

Comment:
I didn't think we were testing repos. Should this not test against repo vulnerabilities?

Reply:
Tests are being written, but may not be part of this test function.